packages/flame/lib/src/components/position_component.dart

```diff
@@ -332,6 +332,10 @@ class PositionComponent extends Component
   /// The absolute center of the component.
   Vector2 get absoluteCenter => absolutePositionOfAnchor(Anchor.center);
 
+  /// Rotates the component to look at given target.
+  void lookAt(Vector2 target) =>
+      angle = Vector2(0, -1).angleToSigned(target - absolutePosition);
```

Comment:
This assumes that the component is "looking up" from the beginning right? I don't think we can assume that unfortunately.

There was another PR a looong time ago setting the "start angle", I'm not sure what happened to that one. I think that was yours too @st-pasha? Did it get down voted for some reason?

Reply:
> This assumes that the component is "looking up" from the beginning right? I don't think we can assume that unfortunately.

Isn't `angle` always measured w.r.t the global up direction?

Reply:
It isn't in regard to anything really, imagine if you have a sprite looking down or to the right for example, then if you would use this method then the angle would be off by 180/90 degrees.

Reply:
In such cases I normally introduce a parent `PositionComponent` for the sprite and rotate that parent by 180/90 in opposite direction so that the sprite points upwards. This feels natural to me coming from other game engines. 

That being said, adding something like `orientation` does feel like a good solution as it helps avoid that additional parent component.

Reply:
Added the suggested `nativeAngle`.